galaxy_ng/app/api/ui/urls.py

```diff
@@ -133,9 +134,14 @@
         'me/',
         viewsets.CurrentUserViewSet.as_view({'get': 'retrieve', 'put': 'update'}),
         name='me'
+    ),
+]
+
+if settings.GALAXY_FEATURE_FLAGS['execution_environments']:
+    paths.append(
+        path('execution-environments/', include(container_paths)),
     )
```

Comment:
In the dev environment, updates to the `GALAXY_FEATURE_FLAGS['execution_environments']` setting auto-reloads the api container and routes are updated as expected. Outside the dev environment I assume a restart of services would be needed for this to take effect.

For unit tests while I can update settings, I cannot find how to reload routes within a test case. If we use this method, it may need to be tested in integration or functional tests.

Reply:
@alikins thanks for the testing suggestion, I confirmed it loaded `PULP_GALAXY_FEATURE_FLAGS` as expected. With the TOML parser the format needed to be: `PULP_GALAXY_FEATURE_FLAGS={execution_environments=false}`